yak/cmake/yak-config.cmake.in

```diff
@@ -0,0 +1,14 @@
+@PACKAGE_INIT@
+
+set(@PROJECT_NAME@_FOUND ON)
+set_and_check(@PROJECT_NAME@_INCLUDE_DIRS "${PACKAGE_PREFIX_DIR}/include")
+set_and_check(@PROJECT_NAME@_LIBRARY_DIRS "${PACKAGE_PREFIX_DIR}/lib")
+
```

Comment:
I recommend removing these. I believe these are only needed for backwards compatibility, but since this is fairly new.

Reply:
I removed these.